ci-operator/config/openstack-k8s-operators/ceilometer-operator/openstack-k8s-operators-ceilometer-operator-main.yaml

```diff
@@ -0,0 +1,38 @@
+base_images:
+  openstack-operator-ci-pre-commit:
+    name: tools
+    namespace: openstack-k8s-operators
+    tag: openstack-operator-ci-pre-commit
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
+    git log -1
+    mkdir -p ../operator && cp -r . ../operator
+    cd ../operator
+    go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+    source <(setup-envtest use -p env)
+    go get golang.stackrox.io/kube-linter/cmd/kube-linter
+    GO111MODULE=on go install golang.stackrox.io/kube-linter/cmd/kube-linter
+    pre-commit run --all-files --show-diff-on-failure
+  container:
+    from: openstack-operator-ci-pre-commit
```

Comment:
You will need to add new pre-commit image like:
https://github.com/openshift/release/blob/ba8a7c3ff6838ea69f548ada6e96412482160281/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L10-L11

With this approach, we can build a pre-commit image with the proper version of golang, defined in the repo config[1].

[1] https://github.com/openstack-k8s-operators/ceilometer-operator/blob/main/.ci-operator.yaml